core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4101,15 +4101,25 @@ public String toString() {
               + "The admin can turn on this property and control where the clients connect in "
               + "the configuration.")
           .build();
-  public static final PropertyKey USER_STREAMING_DATA_TIMEOUT =
-      new Builder(Name.USER_STREAMING_DATA_TIMEOUT)
-          .setAlias("alluxio.user.network.data.timeout.ms", Name.USER_NETWORK_DATA_TIMEOUT)
+  public static final PropertyKey USER_STREAMING_DATA_READ_TIMEOUT =
+      new Builder(Name.USER_STREAMING_DATA_READ_TIMEOUT)
+          .setAlias("alluxio.user.network.data.timeout.ms", Name.USER_NETWORK_DATA_TIMEOUT,
+              Name.USER_STREAMING_DATA_TIMEOUT)
           .setDefaultValue("1h")
```

Comment:
Isn't 1h quite long for both read and write?

Reply:
Yeah, we made the decision to increase the timeouts here (182e08bfe20073f275ed28731401b8dbbf291072) to avoid spurious issues but I agree it is a case by case decision since in certain environments there are benefits to having short timeouts and retrying.

Reply:
For writes, I think a longer timeout is fine because we cannot retry.